core/src/main/java/com/redhat/lightblue/client/response/DefaultLightblueResponse.java

```diff
@@ -0,0 +1,127 @@
+package com.redhat.lightblue.client.response;
+
+import java.io.IOException;
+import java.lang.reflect.Array;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.redhat.lightblue.client.util.JSON;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DefaultLightblueResponse implements LightblueResponse {
```

Comment:
This is the same as the old `LightblueResponse`, I just pulled out an interface for it and renamed the old implementation to `DefaultLightblueResponse`. This is so the top level interface could be immutable (not have setters).